litellm/utils.py

```diff
@@ -7,7 +7,7 @@
 import tiktoken
 
 encoding = tiktoken.get_encoding("cl100k_base")
-import pkg_resources
+import importlib.metadata
```

Comment:
if we use import lib do we need to add it as a dependency in pyptoject ?

Reply:
I don't think so, it's in stdlib:
https://docs.python.org/3/library/importlib.html